.github/workflows/test-prs.yml

```diff
@@ -24,11 +24,7 @@ jobs:
     if: |
       (github.event.pull_request.head.repo.full_name == github.repository) ||
       (github.event.action == 'labeled' && github.event.label.name == 'run-tests')
-    outputs:
-      ref: ${{ steps.get-ref.outputs.ref }}
-      commit: ${{ steps.get-ref.outputs.commit }}
```

Comment:
These were previously needed for e2e testing, but now is no longer needed now that it's being done by the reusable workflow